src/main/java/com/friendly_machines/intellij/plugins/ideanative2debugger/impl/BreakpointManager.java

```diff
@@ -77,15 +78,15 @@ public boolean addBreakpoint(XLineBreakpoint<XBreakpointProperties> key) {
         try {
             Map<String, ?> gdbResponse;
             if (key.isLogMessage()) {
-                gdbResponse = myDebugProcess.dprintfInsert(options.toArray(new String[0]), new String[]{fileLineReference(key.getSourcePosition()), "Breakpointhit"});
+                gdbResponse = myDebugProcess.dprintfInsert(options, List.of(fileLineReference(key.getSourcePosition()), "Breakpointhit"));
             } else {
-                gdbResponse = myDebugProcess.breakInsert(options.toArray(new String[0]), new String[]{fileLineReference(key.getSourcePosition())});
+                gdbResponse = myDebugProcess.breakInsert(options, List.of(fileLineReference(key.getSourcePosition())));
             }
             @SuppressWarnings("unchecked")
             var bkpt = (Map<String, Object>) gdbResponse.get("bkpt");
             myBreakpoints.add(new Breakpoint(myDebugProcess, key, bkpt));
             return true;
-        } catch (GdbMiOperationException | ClassCastException e) {
+        } catch (GdbMiOperationException | ClassCastException | IOException | InterruptedException e) {
```

Comment:
Isn't interruptedException thrown to interrupt threads? Seems very bad to swallow it and not rethrow it. Then the thread wouldn't stop, right?